test/register-bson.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// All test files import BSON from one place here
+// This was born out of debugging our karma configuration
+// But it can also be a convenient way to run the tests against an alternative implementation
+// If you want to run the tests against the typescript directly, change the 'lib' to 'src'
+// and make sure you run mocha using our .mocharc.json or with --require ts-node/register
+
+// This should be done by mocha --require, but that isn't supported until mocha version 7+
+require('chai/register-expect');
```

Comment:
Is this file really about code dupe? If we want to test against typescript then we should just use `ts-node` for everything, right?

Reply:
I wasn't able to get rollup to compile our ts and correctly generate the bundles so the tests make use of the compiled js from typescript because karma relies on rollup to bundle all of mocha into a browser ready bundle